Hide-Chat-Icons/HideChatIcons.plugin.js

```diff
@@ -40,12 +40,6 @@ const React = BdApi.React,
 
 module.exports = class HideChatIcons {
 
```

Comment:
We're moving away from plugin updaters outside of BD entirely, this is why it's gone.